nucleus/security/services/src/main/java/org/glassfish/security/services/provider/authorization/SimpleAuthorizationProviderImpl.java

```diff
@@ -104,18 +102,14 @@ private boolean isDebug() {
     }
 
     @Override
-    public AzResult getAuthorizationDecision(
-        AzSubject subject,
-        AzResource resource,
-        AzAction action,
-        AzEnvironment environment,
-        List<AzAttributeResolver> attributeResolvers ) {
-
-        //TODO: get user roles from Rolemapper, and do the policy  evaluation
-        if ( ! isAdminResource(resource)) {
+    public AzResult getAuthorizationDecision(AzSubject subject, AzResource resource, AzAction action, AzEnvironment environment,
```

Comment:
Next time try to avoid changes in formatting ... or backport them to master too ;-)